apps/web/src/pages/integrations/components/multi-provider/CreateProviderInstanceSidebar.tsx

```diff
@@ -67,8 +75,13 @@ export function CreateProviderInstanceSidebar({
 
   const selectedEnvironmentId = watch('environmentId');
 
-  const showInAppErrorMessage = useMemo(() => {
-    if (!provider || integrations.length === 0 || provider.id !== InAppProviderIdEnum.Novu) {
+  const showNovuProvidersErrorMessage = useMemo(() => {
+    const novuProviders: ProvidersIdEnum[] = [
+      InAppProviderIdEnum.Novu,
+      SmsProviderIdEnum.Novu,
+      EmailProviderIdEnum.Novu,
+    ];
```

Comment:
we have a const in shared package `NOVU_PROVIDERS`

Reply:
Ah nice, let me update it